.travis.yml

```diff
@@ -13,14 +13,14 @@ env:
   - TEST_PLATFORM="DUE"
   - TEST_PLATFORM="LPC1768"
   - TEST_PLATFORM="LPC1769"
-  - TEST_PLATFORM="STM32F1"
+  - TEST_PLATFORM="STM32F103R"
   - TEST_PLATFORM="teensy31"
   - TEST_PLATFORM="teensy35"
   - TEST_PLATFORM="linux_native"
   - TEST_PLATFORM="esp32"
-  - TEST_PLATFORM="alfawise_U20"
-  - TEST_PLATFORM="black_stm32f407ve"
-  - TEST_PLATFORM="adafruit_grandcentral_m4"
+  - TEST_PLATFORM="STM32F103R_alfawise"
```

Comment:
alfawise one is 103V (100 pins with fsmc)

Reply:
I see. So `STM32F103_alfawise` will have to do.